UtopiaCity/Common/Initializers/FlightInitializer.cs

```diff
@@ -27,42 +28,47 @@ public void InitializeSet(ApplicationDbContext context)
             }
             var flight1 = new Flight()
             {
-                FlightNumber = 128,
+                FlightNumber = RandomUtil.GenerateRandomString(200).ElementAt(1),
                 ArrivalTime = DateTime.Now.AddHours(4),
                 DepartureTime = DateTime.Now,
-                Destination = "Moscow,Russia",
+                LocationPoint = "Moscow",
+                DestinationPoint = "Stambul",
                 Weather = "Sunny"
             };
             var flight2 = new Flight()
             {
-                FlightNumber = 256,
+                FlightNumber = RandomUtil.GenerateRandomString(100).ElementAt(2),
                 ArrivalTime = DateTime.Now.AddHours(13),
                 DepartureTime = DateTime.Now.AddHours(1),
-                Destination = "Madrid,Spain",
+                LocationPoint = "Moscow",
+                DestinationPoint = "Madrid",
                 Weather = "Rainy"
             };
             var flight3 = new Flight()
             {
-                FlightNumber = 512,
+                FlightNumber = RandomUtil.GenerateRandomString(150).ElementAt(4),
                 ArrivalTime = DateTime.Now.AddHours(5),
                 DepartureTime = DateTime.Now,
-                Destination = "Stambul,Turkey",
+                LocationPoint="Madrid",
+                DestinationPoint = "Stambul",
                 Weather = "Cloudy"
             };
             var flight4 = new Flight()
             {
-                FlightNumber = 223,
+                FlightNumber = RandomUtil.GenerateRandomString(300).ElementAt(26),
```

Comment:
why elementAt?

Reply:
don't know, first thought was to get more randomly number:), but ok i will change